<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RsaHala</class>
 <widget class="QWidget" name="RsaHala">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>62</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>62</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>62</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>62</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RsaHala</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RsaHala[homedismode=&quot;indoor&quot;] #bsabutton
{
	padding-top:146px;
	background-image: url(:/bithome/image/btn_sa.png);
}
#RsaHala[homedismode=&quot;indoor&quot;] #bsabutton:pressed
{	
	background-image: url(:/bithome/image/btn_sa_click.png);
}
#RsaHala[homedismode=&quot;night&quot;] #bsabutton
{
	padding-top:146px;
	background-image: url(:/bithome/image/btn_sa_night.png);
}
#RsaHala[homedismode=&quot;night&quot;] #bsabutton:pressed
{	
	background-image: url(:/bithome/image/btn_sa_clinight.png);
}

#RsaHala[homedismode=&quot;indoor&quot;] #rsabutton
{
	padding-top:169px;
	background-image: url(:/bithome/image/btn_rtsa.png);
}
#RsaHala[homedismode=&quot;indoor&quot;] #rsabutton:pressed
{	
	background-image: url(:/bithome/image/btn_rtsa_click.png);
}

#RsaHala[homedismode=&quot;night&quot;] #rsabutton
{
	padding-top:169px;
	background-image: url(:/bithome/image/btn_etsa_night.png);
}
#RsaHala[homedismode=&quot;night&quot;] #rsabutton:pressed
{	
	background-image: url(:/bithome/image/btn_rtsa_clinight.png);
}

#RsaHala[homedismode=&quot;indoor&quot;] #ltetddbutton
{
	padding-top:146px;
	background-image: url(:/bithome/image/btn_lte_tdd.png);
}
#RsaHala[homedismode=&quot;indoor&quot;] #ltetddbutton:pressed
{	
	background-image: url(:/bithome/image/btn_lte_tdd_click.png);
}

#RsaHala[homedismode=&quot;night&quot;] #ltetddbutton
{
	padding-top:146px;
	background-image: url(:/bithome/image/btn_tdd_night.png);
}
#RsaHala[homedismode=&quot;night&quot;] #ltetddbutton:pressed
{	
	background-image: url(:/bithome/image/btn_tdd_clinight.png);
}

#RsaHala[homedismode=&quot;indoor&quot;] #ltefddbutton
{
	padding-top:146px;
	background-image: url(:/bithome/image/btn_lte_ftt.png);
}
#RsaHala[homedismode=&quot;indoor&quot;] #ltefddbutton:pressed
{	
	background-image: url(:/bithome/image/btn_lte_ftt_click.png);
}

#RsaHala[homedismode=&quot;night&quot;] #ltefddbutton
{
	padding-top:146px;
	background-image: url(:/bithome/image/btn_fdd_night.png);
}
#RsaHala[homedismode=&quot;night&quot;] #ltefddbutton:pressed
{	
	background-image: url(:/bithome/image/btn_fdd_clinight.png);
}

#RsaHala[homedismode=&quot;indoor&quot;] #pmbutton
{
	padding-top:154px;
	background-image: url(:/bithome/image/btn_pm.png);
}
#RsaHala[homedismode=&quot;indoor&quot;] #pmbutton:pressed
{	
	background-image: url(:/bithome/image/btn_pm_click.png);
}

#RsaHala[homedismode=&quot;night&quot;] #pmbutton
{
	padding-top:154px;
	background-image: url(:/bithome/image/btn_pm_night.png);
}
#RsaHala[homedismode=&quot;night&quot;] #pmbutton:pressed
{	
	background-image: url(:/bithome/image/btn_pm_clinight.png);
}

#RsaHala[homedismode=&quot;indoor&quot;] #interbutton
{
	padding-top:177px;
	background-image: url(:/bithome/image/btn_interfere.png);
}
#RsaHala[homedismode=&quot;indoor&quot;] #interbutton:pressed
{	
	background-image: url(:/bithome/image/btn_interfere_click.png);
}

#RsaHala[homedismode=&quot;night&quot;] #interbutton
{
	padding-top:177px;
	background-image: url(:/bithome/image/btn_interfere_night.png);
}
#RsaHala[homedismode=&quot;night&quot;] #interbutton:pressed
{	
	background-image: url(:/bithome/image/btn_interfere_clinight.png);
}

#RsaHala[homedismode=&quot;indoor&quot;] #fivegbutton
{
	padding-top:154px;
	background-image: url(:/bithome/image/btn_5g.png);
}
#RsaHala[homedismode=&quot;indoor&quot;] #fivegbutton:pressed
{		
	background-image: url(:/bithome/image/btn_5g_click.png);
}

#RsaHala[homedismode=&quot;night&quot;] #fivegbutton
{
	padding-top:154px;
	background-image: url(:/bithome/image/btn_5g_night.png);
}
#RsaHala[homedismode=&quot;night&quot;] #fivegbutton:pressed
{		
	background-image: url(:/bithome/image/btn_5g_click_night.png);
}

#RsaHala[homedismode=&quot;indoor&quot;] #sysbutton
{
	padding-top:154px;
	background-image: url(:/bithome/image/btn_system.png);
}
#RsaHala[homedismode=&quot;indoor&quot;] #sysbutton:pressed
{	
	background-image: url(:/bithome/image/btn_system_click.png);
}

#RsaHala[homedismode=&quot;night&quot;] #sysbutton
{
	padding-top:154px;
	background-image: url(:/bithome/image/btn_system_night.png);
}
#RsaHala[homedismode=&quot;night&quot;] #sysbutton:pressed
{	
	background-image: url(:/bithome/image/btn_system_clinight.png);
}
QPushButton { border-style: flat;color: rgb(238, 238, 236);font: 15pt &quot;Arial&quot;;}

#RsaHala[homedismode=&quot;indoor&quot;] #label
{	
	background-image: url(:/bithome/image/logo_inno_white.png);
}
#RsaHala[homedismode=&quot;night&quot;] #label
{	
	background-image: url(:/bithome/image/logo_inno_night.png);
}

#RsaHala[homedismode=&quot;indoor&quot;] #label_4
{	
	color: rgb(238, 238, 236);
}
#RsaHala[homedismode=&quot;night&quot;] #label_4
{	
	color: rgb(235, 205, 156);
}

#RsaHala[homedismode=&quot;indoor&quot;] #timelb
{	
	color: rgb(238, 238, 236);
}
#RsaHala[homedismode=&quot;night&quot;] #timelb
{	
	color: rgb(235, 205, 156);
}</string>
  </property>
  <property name="homedismode" stdset="0">
   <string>indoor</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="bitresource.qrc">:/bithome/image/top_bar.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="bsabutton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>320</width>
     <height>355</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Spectrum Analyzer</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rsabutton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>320</width>
     <height>355</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Realtime Spectrum
Analyzer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ltetddbutton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>320</width>
     <height>355</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LTE-A TDD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ltefddbutton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>90</y>
     <width>320</width>
     <height>355</height>
    </rect>
   </property>
   <property name="text">
    <string>LTE-A FDD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pmbutton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>445</y>
     <width>320</width>
     <height>355</height>
    </rect>
   </property>
   <property name="text">
    <string>Power Meter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="interbutton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>445</y>
     <width>320</width>
     <height>355</height>
    </rect>
   </property>
   <property name="text">
    <string>Interference
Analyzer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fivegbutton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>445</y>
     <width>320</width>
     <height>355</height>
    </rect>
   </property>
   <property name="text">
    <string>5G Analyzer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sysbutton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>445</y>
     <width>320</width>
     <height>355</height>
    </rect>
   </property>
   <property name="text">
    <string>System Setting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>19</y>
     <width>146</width>
     <height>53</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="timelb">
   <property name="geometry">
    <rect>
     <x>1031</x>
     <y>24</y>
     <width>146</width>
     <height>48</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1187</x>
     <y>26</y>
     <width>53</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="bitresource.qrc">:/bithome/image/icn_home_charge_5.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1127</x>
     <y>56</y>
     <width>108</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Language</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bitresource.qrc"/>
 </resources>
 <connections/>
</ui>
